RecoBTag/TensorFlow/test/test_deep_doubleb_cfg_AOD.py

```diff
@@ -62,7 +62,7 @@
    process,
    #labelName = 'AK8PFCHS',
    labelName = 'AK8PFPuppi',
-   #jetSource = cms.InputTag('ak8PFJetsCHS'),
+   #jetSource = cms.InputTag('ak8PFJetsPuppi'),
```

Comment:
this was marked as resolved at the point when modifications in other test files (PhysicsTools/PatAlgos/test/patTuple_addBTagging_cfg.py PhysicsTools/PatAlgos/test/patTuple_addJets_cfg.py) were removed/reverted , however the file is still modified, even though in a more harmless way.
Was it intentional to keep?

Reply:
Yes, having this as the "test" case is desired.